Assets/HoloToolkit/UX/Scripts/Buttons/Button.cs

```diff
@@ -240,7 +247,7 @@ public void OnFocusEnter(PointerSpecificEventData eventData)
         public void OnFocusExit(PointerSpecificEventData eventData)
         {
              if (!m_disabled) // && FocusManager.Instance.IsFocused(this))
-            {
+             {
```

Comment:
Extra space here (there might be one on the above `if` statement, which is what was throwing this line off).

Reply:
Doesn't the original line have a missing space? I just thought adding the space makes it conform.